processor/transform/single-code-tabs.js

```diff
@@ -0,0 +1,18 @@
+import visit from 'unist-util-visit';
```

Comment:
I really don't like that each tiny utility is it's own package. It was my understanding the momentum was moving away from this kind of architecture. This was recommended for awhile to prevent huge imports, but now we have tree shaking. And I'm terrible at keeping the deps up to date, and the more packages we install, the more annoying I find it.